request-processor/docker-entrypoint.sh

```diff
@@ -1,4 +1,4 @@
 #!/bin/sh
 set -e
 echo "${DATABASE_URL}"
-celery --config celeryconfig -A tasks worker -l  INFO  -P eventlet --concurrency ${CELERY_WORKER_CONCURRENCY:-1}
+celery --config celeryconfig -A tasks worker -l  INFO  -P eventlet --concurrency ${CELERY_WORKER_CONCURRENCY:-2}
```

Comment:
Do we really want to change the default concurrency level at this point?  I think not!